configure

```diff
@@ -579,10 +580,6 @@ def configure_arm(o):
     o['variables']['arm_fpu'] = 'vfpv3'
     o['variables']['arm_version'] = '7'
 
-  # Print warning when snapshot is enabled and building on armv6
-  if is_arch_armv6() and options.with_snapshot:
-    warn('when building on ARMv6, don\'t use --with-snapshot')
-
```

Comment:
pi1-raspbian-wheezy at least is armv6.

@rvagg can we get those renamed at som point to follow the naming schema `arch-distro-version` like the rest of the CI jobs, to avoid confusion.